qa/tasks/salt_util.py

```diff
@@ -0,0 +1,150 @@
+'''
+Salt utility module
+
+Usage: First, ensure that there is a role whose name corresponds
+to the value of the master_role variable, below. Second, in your
+task, instantiate a SaltUtil object:
+
+    from salt_util import SaltUtil
+
+    su = SaltUtil(ctx, config)
+
+Third, enjoy the SaltUtil goodness - e.g.:
+
+    su.ping_minions()
+
+'''
+import logging
+import re
+
+from cStringIO import StringIO
+from teuthology import misc
+from teuthology.contextutil import safe_while
+from teuthology.exceptions import (CommandFailedError, ConfigError)
+from teuthology.orchestra.remote import Remote
+from teuthology.orchestra import run
+
+log = logging.getLogger(__name__)
+master_role = 'client.salt_master'
+
+
+class SaltUtil(object):
```

Comment:
Since SaltUtil capture some context it is better to rename it to SaltCtl|Controller or SaltMgr|Manager?

Reply:
OK, I will change it to SaltManager.